index.php

```diff
@@ -8,4 +8,9 @@
 // Startup
 require_once(DIR_PUBLIC . '/system/startup.php');
 
-start(APPLICATION);
\ No newline at end of file
+start(APPLICATION);
+
+// Output
+global $response;
+$response->setCompression($config->get('config_compression'));
```

Comment:
I have "bad"  experience with compression. On my server, even with compression set to "0", OC was sending garbage data. Just for the note, if I see that again: https://github.com/opencart/opencart/issues/4832